src/Language/PureScript/Ide/State.hs

```diff
@@ -194,11 +196,11 @@ populateStage2 :: (Ide m, MonadLogger m) => m ()
 populateStage2 = do
   st <- ideStateVar <$> ask
   duration <- liftIO $ do
-    start <- getCurrentTime
+    start <- getTime Monotonic
     atomically (populateStage2STM st)
```

Comment:
Doesn't this function just fork a thread? In which case, you'd be measuring the wrong thing here, I think.